locale/cs/LC_MESSAGES/clipboard-indicator.po

```diff
@@ -59,27 +58,27 @@ msgstr "Následující položka"
 
 #: prefs.js:119
 msgid "History Size"
-msgstr "Velikost historie"
+msgstr "Hloubka historie"
 
 #: prefs.js:124
 msgid "Refresh Interval (ms)"
-msgstr "Interval obnovení (ms)"
+msgstr "Interval obnovování (ms)"
 
 #: prefs.js:129
 msgid "Preview Size (characters)"
 msgstr "Velikost náhledu (znaky)"
 
 #: prefs.js:134
 msgid "Max cache file size (kb)"
-msgstr "Maximální velikost mezipaměti (kb)"
+msgstr "Omezit velikost mezipaměti na nejvýše (KB)"
```

Comment:
What's wrong with the lowercase kb?

Reply:
Lowercase b is not correct for this case – kilobit vs kilobyte:
https://en.wikipedia.org/wiki/Kilobit
https://en.wikipedia.org/wiki/Kilobyte
correct form would be kB, but it is practically not used anywhere, most common is using prefix as upper case letter. So KB, MB, GB (or KiB, MiB, GiB for binary).